java-numbers-2/src/main/java/com/baeldung/lcm/SimpleAlgorithm.java

```diff
@@ -0,0 +1,16 @@
+package com.baeldung.lcm;
+
+public class SimpleAlgorithm {
+    public static int lcm(int number1, int number2) {
+        if (number1 == 0 || number2 == 0) {
+            return 0;
+        }
+        int absNumber1 = Math.abs(number1);
+        int absNumber2 = Math.abs(number2);
+        int lcm = Math.max(absNumber1, absNumber2);
+        while (lcm % Math.min(absNumber1, absNumber2) != 0) {
```

Comment:
Observation: absNumber1 and absNumber2 do not change inside the loop. It would be more efficient to store the Math.min and Math.max results in local variables before entering the loop, rather than invoking these methods in each loop iteration.

Reply:
@KevinGilmore Thanks for pointing this out. I have changed it now.